app/OwnsModels.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+trait OwnsModels
+{
+    /**
+     * Determine if this model owns the given model.
+     *
+     * @param Model $model
+     *
+     * @return bool
+     */
+    public function owns(Model $model)
+    {
+        return $this->id == $model->{$this->getForeignKey()};
```

Comment:
Mhm. Yah, if it's an integer primary key, I'm a big fan of making it always present as an integer. Friggin PHP.